docs-src/go/how-to-schedule-a-workflow-in-go.md

```diff
@@ -0,0 +1,233 @@
+title: How to Schedule a Workflow in Go 
+sidebar_label: Schedule
+description: Set a Schedule in Go for scheduling Workflows to run at specific times.
+tags:
+  - go
+  - developer-guide
+  - how-to
+---
+
+Scheduling [Workflows](/concepts/what-is-a-workflow) is crucial to automation.
+By scheduling [Workflow Executions](/concepts/what-is-a-workflow-execution), you can reduce manual intervention and ensure timely execution of business processes.
+
+Temporal allows you to execute multiple Schedule operations, either through code or with the [CLI tool](/concepts/what-is-the-temporal-cli).
+Read on to find out how to create and edit Schedules.
+
+:::note
+Be sure to [enable Schedules in your environment](/concepts/what-is-a-schedule#limitations) before creating any Schedules.
+:::
+
+## Create
+
+Schedules are created with the `create` action.
+For each new Schedule, tbe Temporal Server generates a unique Schedule ID.
+
+To create a Schedule in Go, use `ScheduleClient().Create()` on the [Client](/concepts/what-is-the-temporal-client).
+Schedules must be initialized with a Schedule ID, [Spec](/concepts/what-is-a-schedule#spec), and [Action](/concepts/what-is-a-schedule#action) to perform.
+Enter these values in `client.Schedule.Options{}`.
+
+<!--SNIPSTART samples-go-schedule-create-delete {"selectedLines": ["26-38"]}-->
+
+[schedule/starter/main.go](https://github.com/temporalio/samples-go/blob/master/schedule/starter/main.go)
+
+```go
+// This schedule ID can be user business logic identifier as well.
+scheduleID := "schedule_" + uuid.New()
+workflowID := "schedule_workflow_" + uuid.New()
+// Create the schedule, start with no spec so the schedule will not run.
+scheduleHandle, err := c.ScheduleClient().Create(ctx, client.ScheduleOptions{
+	ID:   scheduleID,
+	Spec: client.ScheduleSpec{},
+	Action: &client.ScheduleWorkflowAction{
+		ID:        workflowID,
+		Workflow:  schedule.SampleScheduleWorkflow,
+		TaskQueue: "schedule",
+	},
+})
+if err != nil {
+	log.Fatalln("Unable to create schedule", err)
+}
+```
+
+<!--SNIPEND-->
+
+## Backfill
+
+Backfilling a Schedule executes [Workflow Tasks](/concepts/what-is-a-workflow-task) ahead of the Schedule's specified time range.
+This is useful when you need to execute a missed or delayed Action, or if you want to test the Workflow ahead of time.
+
+To backfill a Schedule in Go, use `Backfill()` on `scheduleHandle`.
+
+## Delete
+
+Deleting a Schedule erases a Schedule from a Workflow.
+Deletion does not affect any Workflows started by the Schedule.
+
+To delete a Schedule in Go, use the `Delete()` command on `scheduleHandle`.
+
+<!--SNIPSTART samples-go-schedule-create-delete {"selectedLines": ["42-49"]}-->
+
+[schedule/starter/main.go](https://github.com/temporalio/samples-go/blob/master/schedule/starter/main.go)
+
+```go
+// Delete the schedule once the sample is done
+defer func() {
+	log.Println("Deleting schedule", "ScheduleID", scheduleHandle.GetID())
+	err = scheduleHandle.Delete(ctx)
```

Comment:
With these samples showing how to use a `scheduleHandle`, we should show them how to obtain it (using the one from create is often not a realistic use even though we do it in the sample).

Reply:
> How do we obtain it?

Via `ScheduleClient.GetHandle`

> I am working from what was done in code

You are working with what was done in a single sample that creates and deletes in the same bit of code. But our docs shouldn't expect people would use it that way. Sample code and docs code can often have mismatched purposes. In this case, samples show you how to do everything together at once and docs separate them into standalone sections. These are incompatible in this case.

> If there's another place mentioning the use of scheduleHandle, please let me know

You may have to write it.

Reply:
Very well. I'm going to go ahead and put in code blocks in lieu of the code snippets. Once I migrate this over to dacx format, then I'll write code for a use case.

Reply:
May want to confirm with @flossypurse. Maybe updating the sample to break out the steps makes more sense. That's what was done in the Python schedule sample.